src/libostree/ostree-repo-commit.c

```diff
@@ -1736,7 +1740,7 @@ ostree_repo_prepare_transaction (OstreeRepo     *self,
     return FALSE;
 
   /* Success: do not abort the transaction when returning. */
-  txn = NULL; (void) txn;
+  txn->repo = NULL; (void) txn;
```

Comment:
Minor nit, I think the `(void) txn;` is now unnecessary because we're performing an active assignment from the PoV of the compiler.